Gruntfile.js

```diff
@@ -223,7 +224,8 @@ module.exports = (grunt) => {
         'process.env.MAX_EVENTS_AFTER': `'${MAX_EVENTS_AFTER}'`,
         'process.env.NODE_ENV': `'${NODE_ENV}'`,
         'process.env.EXPOSE_SBP': `'${EXPOSE_SBP}'`,
-        'process.env.ENABLE_UNSAFE_NULL_CRYPTO': `'${ENABLE_UNSAFE_NULL_CRYPTO}'`
+        'process.env.ENABLE_UNSAFE_NULL_CRYPTO': `'${ENABLE_UNSAFE_NULL_CRYPTO}'`,
+        'process.env.UNSAFE_TRUST_ALL_MANIFEST_SIGNING_KEYS': UNSAFE_TRUST_ALL_MANIFEST_SIGNING_KEYS
```

Comment:
Inconsistent handling of this environment variable, could you please wrap it the same way as the others?

Reply:
but this is a boolean, not a string

Reply:
So is ENABLE_UNSAFE_NULL_CRYPTO, I don't think it matters

Reply:
It does matter (although I can see the argument for consistency). See:

```js
if (process.env.ENABLE_UNSAFE_NULL_CRYPTO === 'true' && ...
```

If you make it a string, then it's a string. If you don't quote it, it's whatever the type of that expression is.

Reply:
I see. Well, can we make it consistent anyway, for consistency's sake?